tox.ini

```diff
@@ -33,6 +33,16 @@ commands =
   python -m sphinx -v -b doctest -d doctrees docs html
   find html -type f -name "*.ipynb" -not -path "html/_sources/*" -delete
 
+[testenv:docs-clean]
+description = Clean all the existing built objects and build them from scratch
+allowlist_externals =
+  find
+  tox
+commands =
+  find {toxinidir}/doctrees ! -name doctrees -delete
+  find {toxinidir}/html ! -name html -delete
+  tox -e docs
```

Comment:
Is a nested call to `tox` recommended? Is there a better way? If not, why not have users call `tox -e docs-clean` and then `tox -e docs` by hand?